src/functions/module_statistical.ts

```diff
@@ -555,6 +772,99 @@ export const COVARIANCE_S = {
   isExported: true,
 } satisfies AddFunctionDescription;
 
+// -----------------------------------------------------------------------------
+// FORECAST
+// -----------------------------------------------------------------------------
+export const FORECAST: AddFunctionDescription = {
+  description:
+    "Calculates the expected y-value for a specified x based on a linear regression of a dataset.",
+  args: [
+    arg("x (number)", _t("The value on the x-axis to forecast.")),
+    arg(
+      "data_y (range<number>)",
+      _t("The range representing the array or matrix of dependent data.")
+    ),
+    arg(
+      "data_x (range<number>)",
+      _t("The range representing the array or matrix of independent data.")
+    ),
+  ],
+  returns: ["NUMBER"],
+  compute: function (
+    x: Maybe<CellValue>,
+    dataY: Matrix<CellValue>,
+    dataX: Matrix<CellValue>
+  ): number {
+    const _dataY = matrixMap(dataY, (value) => toNumber(value, this.locale));
+    const _dataX = matrixMap(dataX, (value) => toNumber(value, this.locale));
+    return predictLinearValues(_dataY, _dataX, toMatrix(toNumber(x, this.locale)), true)[0][0];
+  },
+  isExported: true,
+};
+
+// -----------------------------------------------------------------------------
+// GROWTH
+// -----------------------------------------------------------------------------
+export const GROWTH: AddFunctionDescription = {
+  description: "Fits points to exponential growth trend.",
+  args: [
+    arg(
+      "known_data_y (range<number>)",
+      _t(
+        "The array or range containing dependent (y) values that are already known, used to curve fit an ideal exponential growth curve."
+      )
+    ),
+    arg(
+      "known_data_x (range<number>, optional, default = {1,2,3,...})",
```

Comment:
```Optional``` can be removed because ```default``` implies it

currently the spaces before and after the equal of the default value do not allow the transcription of the information in the user interface. Maybe it's better for the moment to do ```default={1,2,3,...}```

Also 
also, I see that the use of double brace ```{....}" is not supported either.